relentless/ensemble.py

```diff
@@ -8,14 +8,59 @@
 from . import core
 
 class RDF(core.Interpolator):
+    """Constructs a :py:class:`Interpolator` object interpolating through an RDF
+    function :math:`g(r)`, and creates an `nx2` array with columns as `r` and `g`.
+
+    Parameters
+    ----------
+    r : array_like
+        1-d array of r values, that must be continually increasing.
+    g : array_like
+        1-d array of g values.
+
+    """
     def __init__(self, r, g):
         super().__init__(r,g)
         self.table = np.column_stack((r,g))
 
 class Ensemble(object):
-    def __init__(self, types, T, P=None, V=None, mu={}, N={}, kB=1.0, conjugates=None):
-        self.types = tuple(types)
-        self.rdf = core.PairMatrix(self.types)
+    """Constructs a thermodynamic ensemble for a set of types with specified parameters.
```

Comment:
```suggestion
    """Thermodynamic ensemble.
```
Let's also add some details here about how ensembles are defined. This should include that T is always a constant, then either P or V and either mu or N for each type can be specified. Also, that the types are the ones that are in the mu or N dicts.